Windows/sgx/libsgx_tsgxssl/tw_critical_sec.cpp

```diff
@@ -29,7 +29,6 @@
  *
  */
 
-#include "libsgx_tsgxssl_t.h"
 #include "tCommon.h"
```

Comment:
Should be `tcommon.h`?

Reply:
Technically it should be, but was not a real problem in Windows. I can fix it.